sig-service-catalog/charter.md

```diff
@@ -0,0 +1,169 @@
+# SIG Service Catalog Charter
+
+This charter adheres to the conventions described in the [Kubernetes Charter
+README](https://github.com/kubernetes/community/blob/master/committee-steering/governance/README.md).
+
+## Scope
+
+Service Catalog is a Kubernetes extension project that implements the [Open
+Service Broker API](https://www.openservicebrokerapi.org/) (OSBAPI). It allows
```

Comment:
I disagree with this scope. People can consume cloud services natively in Kubernetes without service broker.

How about something like:

It enables application developers to provision cloud services from within Kubernetes and integrates configuration and discovery of those services into Kubernetes resources.

Reply:
I can live with that rewording

Reply:
Fixed